examples/bilingual_spanish_english_kindle/en_or_es_metadata_parser.py

```diff
@@ -0,0 +1,11 @@
+from .en_metadata_parser import en_metadata_parser
+from .es_metadata_parser import es_metadata_parser
+import re
+
+
+def en_or_es_metadata_parser(metadata_line: str):
+    """Parse @metadata_line in Spanish or English"""
+    if re.search("Your", metadata_line):
```

Comment:
Ah that's true! TIL the difference between `re.match` and `re.search` - cheers. I think in this case `re.search` is warranted, since only the metadata line is searched (see `parser.py/parse_clippings()`) and the match string would need to be changed to `"- Your"` for lines of form `"- Your Highlight on page 115 | location 1756-1757 | Added on Friday, 26 February 2021 11:03:02"`.